roles/validate_http_store/defaults/main.yml

```
@@ -1,3 +1,4 @@
 http_store_dir : "{{ iso_download_dest_path | default('/opt/http_store/data') }}"
-http_host: "{{ discovery_iso_server | default('http://' + hostvars['http_store']['ansible_host']) }}"
+http_port: 80
+http_host: "{{ discovery_iso_server | default('http://' + hostvars['http_store']['ansible_host']) }}:{{ http_port }}"
```

You need to put the http_port into the default otherwise it will get added twice as it would have to be in `discovery_iso_server`
```suggestion
http_host: "{{ discovery_iso_server | default('http://' + hostvars['http_store']['ansible_host'] + ':' + http_port') }}"
```